release/golden_notebook_tests/workloads/dask_xgboost_test.py

```diff
@@ -9,15 +10,9 @@
 FILE_URL = "https://archive.ics.uci.edu/ml/machine-learning-databases/" \
            "00280/HIGGS.csv.gz"
 
-parser = argparse.ArgumentParser()
-parser.add_argument(
-    "--smoke-test", action="store_true", help="Finish quickly for testing.")
-args = parser.parse_args()
-
-
-def main():
-    ray.client("anyscale://").connect()
 
+@ray.remote
+def test():
```

Comment:
Is wrapping in a task necessary for this to work? We shouldn't be wrapping in a task here otherwise we won't actually be testing whether the script works with Anyscale connect.